src/main/resources/META-INF/rust-core.xml

```diff
@@ -1115,6 +1117,10 @@
                 class="org.rust.cargo.runconfig.command.RunCargoCommandAction"
                 text="Run Cargo Command"/>
 
+        <action id="Cargo.RunWasmpackCommand"
```

Comment:
I could suggest to change it to `Cargo.RunWasmPackCommand` as it supposed to be. Then there would be no typo error anymore.

Back in time, I was also thinking about the choice between `Wasmpack` and `WasmPack` in the class names. Because of the `wasm-pack` and `WasmPack` writing, and already existing `wasm` word in the dictionary may be right now there is no need to add it there.

Anyway, below I see we decided to remove this action.

Reply:
Decided to add it because of the current package name `wasmpack`.

<img width="879" alt="image" src="https://user-images.githubusercontent.com/6342851/89509299-d170ea00-d7d7-11ea-8544-babd0c626220.png">